### ISSUE TITLE
XRTOS Hangs on uC/OS-II if task is not created with extended information

### ISSUE THREAD

**Issue by CIPop** [Marus/cortex-debug#797]
The following code will cause the debugger to hang once a breakpoint is used after this task is created:

```C
err = OSTaskCreate(Consume, (void*)0, &Task3Stk[APP_CFG_TASK_START_STK_SIZE-1], APP_TASK_TEST3_PRIO);
```

Replacing this code with the extended version which populates stack information will work as expected:

```C
  err = OSTaskCreateExt(
    Consume, 
    (void*)0, 
    &Task3Stk[APP_CFG_TASK_START_STK_SIZE-1], 
    APP_TASK_TEST3_PRIO, 
    APP_TASK_TEST3_PRIO, 
    &Task3Stk[0], 
    APP_CFG_TASK_START_STK_SIZE,
    NULL, 
    OS_TASK_OPT_STK_CHK);
```

Getting the following output with the workaround:
<img width="1274" alt="Screenshot 2023-01-07 at 6 58 34 PM" src="https://user-images.githubusercontent.com/8635911/211178500-2e4c95b3-8bba-4822-8f9e-05becd77d19f.png">

**Expected** When information isn't provided, the debugger should not attempt to retrieve it.
**Actual** The debugger hangs for a few seconds, not accepting any further commands.

I tried disabling OpenOCD RTOS in `launch.json` (`//            "rtos": "auto",`) but it did not have any effect.

**Comment by CIPop** [comment]
Immediate workaround if `OSTaskCreateExt` can't be used for any reasons: disable XRTOS by unchecking the option in VSCode. (Removing the OpenOCD RTOS parameter won't have any effect.)

<img width="162" alt="Screenshot 2023-01-07 at 7 06 52 PM" src="https://user-images.githubusercontent.com/8635911/211178689-c1307064-0251-4e37-b136-1eef8ea6acf9.png">

**Comment by PhilippHaefele** [comment]
Thanks for reporting the issue. Please note that the development of the RTOS View has moved to https://github.com/mcu-debug/rtos-views

Seems i didn't have foreseen the case with `OS_TASK_CREATE_EXT_EN` being enabled, but not using it.

Will have a look 👨‍💻

**Comment by CIPop** [comment]
Thank you @PhilippHaefele ! That was very fast :).

I will make sure to report any new issues in the correct repo next time.

**Comment by PhilippHaefele** [comment]
No problem and thanks to @haneefdm a released version 0.0.3 is already available from the marketplace. Have fun 👨‍💻 

> I will make sure to report any new issues in the correct repo next time.

Thanks 😃
